samples-generator/lib/createBuildingsTile.js

```diff
@@ -189,3 +225,46 @@ function generateBatchTableBinary(buildings) {
         binary : batchTableBinary
     };
 }
+
+function generateBatchTableBinary(buildings) {
+    var buildingsLength = buildings.length;
+    var cartographicBuffer = Buffer.alloc(buildingsLength * 3 * sizeOfDouble);
+    var codeBuffer = Buffer.alloc(buildingsLength * sizeOfUint8);
+
+    var batchTableJson = {
+        cartographic : {
+            name : 'CartographicBuffer',
```

Comment:
For consistency with the legacy batch table names rename these properties:
* `CartographicBuffer` -> `cartographic`
* `Code` -> `code`